packages/referral_reconciliation/test/referral_reconciliation_test.dart

```diff
@@ -0,0 +1 @@
+void main() {}
```

Comment:
Please add the necessary Test cases

Reply:
Test cases will be added in [HLM-5567](https://digit-discuss.atlassian.net/browse/HLM-5567)